lib/CmdHandler/CmdHandler.cpp

```diff
@@ -140,7 +142,7 @@ void CmdHandler::task() {
         carState.MotorCurrent = floatValue;
         break;
       case '-':
-        carControl.adjust_paddles(10);
+        carControl.adjust_paddles(3);
```

Comment:
What is this value about? Maybe a constant in a class/parameter?

Reply:
This is the time (about) waiting for paddle input at adjusting. Same in boot sequence but longer.
If you has a good place we can migrate it

Reply:
Well, we can surely let it here, but I suggest to add ad comment that this is the wait time.